src/use-swr.ts

```diff
@@ -24,6 +25,10 @@ import {
 
 type Revalidator = (...args: any[]) => void
 
+// Generate strictly increasing timestamps.
+let __timestamp = 0
```

Comment:
do you feel it's okay to start timestamp from `1` to avoid some issue like `!now()` ?

Reply:
:+1: (where are we using `!now()` btw?)

Reply:
we used to have this issue in #1075

Reply:
I just checked again, we are using `const now = () => ++__timestamp` so it will be starting from 1 I think.